src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -330,10 +330,18 @@ public ICommand SetParameterValue(IParameter parameter, double value, bool shoul
 
       private IPKSimCommand commandForRelativeExpressionParameter(IParameter parameter, double value)
       {
-         if (!parameter.IsExpressionNorm())
-            return new SetRelativeExpressionInSimulationAndNormalizedCommand(parameter, value);
+         //Normalized expression parameters are readonly and will be updated when setting relative expression
+         if (parameter.IsExpressionNorm())
+            return new PKSimEmptyCommand();
```

Comment:
This is the big change here. If a normalized parameter value is set (which can happen when trying to update dyanmically one container from another, for example when update or commit is performed), then we do not do anything

Reply:
the idea is that the rel exp will also be changed and will update the rel exp norm anyways. Previously, it was  updating both ways resulting in very weird outcome

Reply:
the command `SetRelativeExpressionFromNormalizedCommand` is therefore not use anywhere. We might as well delete it at some point  (setting relexp from norm values)